lib/shift.rb

```diff
@@ -0,0 +1,44 @@
+class Shift
```

Comment:
I'm wondering if this can be implemented in a module.  on second look, it doesn't seem like there is much use for a shift class, ultimately I only care about the hash that is returned from  the total_shifts method. This all currently works but should be reevaluated in a later iteration once this one is pulled in and the other required functionality is up and running.